### DISCUSSION TITLE
Is there any APIs to send USDC from Phantom Wallet to a DEX platform directly?

### DISCUSSION THREAD

**Discussion by abolfazl-sharifi** [phantom/docs#138] [General]
We can send Solana from Phantom Wallet to a DEX platform. But, we have issue for sending USDC. Is there any sample or APIs to send it?

**Comment by adamdelphantom** [comment]
Hi @abolfazl-sharifi,

This article from QuickNode is a good starting place for sending any SPL token. Once you have the transaction built you can use our docs for sending the transaction in phantom. https://docs.phantom.app/solana/integrating-phantom

**Reply by abolfazl-sharifi** [reply]
Hi @adamdelphantom - Thank you so much.
